spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonDeserializer.java

```diff
@@ -97,12 +97,14 @@
 	public static final String USE_TYPE_INFO_HEADERS = "spring.json.use.type.headers";
 
 	/**
-	 * A method name to determine the {@link JavaType} to deserialize the key to.
+	 * A method name to determine the {@link JavaType} to deserialize the key to:
+	 * 'com.Foo.deserialize'. See {@link JsonTypeResolver#resolveType} for the signature.
```

Comment:
Can we avoid using `foo/bar` in the production code and docs?
Thanks

Reply:
Thought that it's used intentionally in the project - see `TYPE_MAPPINGS` above:
```java
	/**
	 * Kafka config property to add type mappings to the type mapper:
	 * 'foo=com.Foo,bar=com.Bar'.
	 */
```